packages/components/src/components/es-empty-state/es-empty-state.tsx

```diff
@@ -0,0 +1,34 @@
+import { Component, h, Host, Prop, type VNode } from '@stencil/core';
+import type { EmptyStateLayout } from './types';
+
+/**
+ * Display an empty state with an illustration and a message.
+ */
+@Component({
+    tag: 'es-empty-state',
+    styleUrl: 'es-empty-state.css',
+    shadow: true,
+})
+export class EmptyState {
+    /** The header of the empty state. */
+    @Prop() header!: string;
+    /** The body of the empty state. */
+    @Prop() body!: string | VNode;
```

Comment:
This is kind of weird, I think it was done like this to add a link with minimal changes in cloud. 

Maybe we should take this opportunity to get rid of this prop, and replace:

```tsx
<p class={'body'}>{this.body}</p>
```

with 

```tsx
<p class={'body'}>
    <slot />
</p>
```

Reply:
(So then the usage becomes:

```tsx
    <y-empty-state header={'Create a new group'}>
        <es-illustration-group slot={'illustration'} />
        {'Creating a new group will allow for the grouping of members with the same access.'}
        <es-button-link
            slot={'foot'}
            variant={'outline'}
            url={'/new'}
        >
            {'New group'}
            <es-icon icon={'plus'} slot={'after'} />
        </es-button-link>
    </y-empty-state>
```

)